selinux/bluechi.te

```diff
@@ -57,7 +58,7 @@ kernel_dgram_send(bluechi_t)
 logging_send_syslog_msg(bluechi_t)
 logging_read_syslog_pid(bluechi_t)
 
-allow haproxy_t bluechi_t:unix_stream_socket connectto;
```

Comment:
I think the haproxy_t thing was needed for the TLS proxy work we demonstrated at some point.

Reply:
Ah, i see you just moved it.

Reply:
Yes, I wanted to group everything haproxy related. Sorry for the confusion.